srv/src/main/java/my/bookshop/handlers/AdminServiceHandler.java

```diff
@@ -94,14 +94,6 @@ public void beforeCreateOrder(Stream<Orders> orders, EventContext context) {
 				order.getItems().forEach(orderItem -> {
 					// validation of the Order creation request
 					Integer quantity = orderItem.getQuantity();
-					if (quantity == null || quantity <= 0) {
-						// errors with localized messages from property files
-						// exceptions abort the request and set an error http status code
-						// messages in contrast allow to collect multiple errors
-						messages.error(MessageKeys.QUANTITY_REQUIRE_MINIMUM)
```

Comment:
The message removal seems to be missing from this PR.

Reply:
The message is still used [here](https://github.com/SAP-samples/cloud-cap-samples-java/pull/402/files?diff=unified&w=0#diff-af608d1373e020896c7b3dfc5904d6e45e8e69219e1efa995396a5f5b060bb25L207). I think this is the warning, which is displayed on the UI. Can we get rid of this as well?

Reply:
Ah good catch, I guess that can't be easily removed, if we want to keep the warning..